share/doc/methods.rst

```diff
@@ -334,19 +334,22 @@ Iterative methods
 =================
 
 The following sections deal with the methods of Iterative Boltzmann
-Inversion (IBI), Inverse Monte Carlo (IMC), and Relative Entropy (RE).
+Inversion (IBI), Inverse Monte Carlo (IMC), Iterative Integral
```

Comment:
Since it's a small family of methods, it'd sound rather strange to me.

Reply:
Ya, I think your right, but there is something grammatically funky, might just be how the methods are named, because this method is called an Equation. If you want to make it clean, I would change it so it was like this:

The following sections deal with the Iterative Boltzmann Inversion (IBI) method, the Inverse Monte Carlo (IMC) method, the Iterative Integral Equation (IIE) method, and the Relative Entropy (RE) method. 

It's a bit redundant but I think grammatically it is a little better.